### ISSUE TITLE
Private members have implicit any type in generated declaration files

### ISSUE THREAD

**Issue by olee** [microsoft/TypeScript#37621]
It seems the bug that was fixed in #36216 is happening again:
When generating delcarations for private class members, no type information is appended which causes the declaration file to generate errors when imported into a project with noImplicitAny enabled.

**TypeScript Version:**  3.8.3

**Code**

```ts
// Test.ts
export default class Test {
    private _number: number = 0;
    public str: string = '';
    private get num() { return this._number; }
}
```

**Expected behavior:**
```ts
// Test.d.ts
export default class Test {
    private num: any;
    str: string;
    private get num(): any;
}
```

**Actual behavior:**
```ts
// Test.d.ts
export default class Test {
    private _number;
    str: string;
    private get num();
}
```

**Related Issues:** #36216, #36639

**Comment by sandersn** [comment]
The version without `any` is the intended emit: `private _number`. But you're not supposed to get an implicit any error with that file.

When I manually write and compile that .d.ts file, I do not get an error. Are you using a version older than 3.8.3? Can you give more details about your setup? A standalone repo would be most helpful.

**Comment by olee** [comment]
Yes, the project consuming the .d.ts file is still running on older typescript version.
From what I read in the referenced issues, I thought it was changed to append the any type to the declarations, but after checking again I see that the resolution was actually to just ignore implicit any errors in .d.ts files. Is that correct?

If so, I guess this issue is invalid and there's no other way but to try upgrading the other typescript project.

**Comment by sandersn** [comment]
Re-reading #35216, it looks like 3.6 and 3.7 got patch versions. I take it you are working with a version older than that?
